docs/uml-class-diagrams/display02/test/DisplayComm2WayInterfaceImplementationTests/DisplayComm2WayInterfaceImplementationTests.puml

```diff
@@ -2,8 +2,10 @@
 
 package edu.oakland.test.display02 {
     class DisplayComm2WayInterfaceImplementationTests {
-
+        + void rfidInIsInt()
```

Comment:
You know that this will be true already, or JVM would have thrown an error. You can thus remove this test method (and save yourself some work).
```suggestion
```